connector_elasticsearch/models/se_backend_elasticsearch.py

```diff
@@ -33,3 +36,27 @@ def _server_env_fields(self):
         env_fields = super()._server_env_fields
         env_fields.update({"es_server_host": {}})
         return env_fields
+
+    def action_test_connection(self):
+        with self.specific_backend.work_on(self._name) as work:
+            adapter = work.component(usage="se.backend.adapter")
+            es = adapter._get_es_client()
+            try:
+                es.security.authenticate()
+            except NotFoundError:
+                raise UserError(_("Unable to reach host."))
+            except AuthenticationException:
+                raise UserError(_("Unable to authenticate. Check credentials."))
+            except Exception as e:
+                raise UserError(
+                    _("Unable to connect to ElasticSearch:") + "\n\n" + repr(e)
```

Comment:
Is it safe to use repr here ?

Reply:
Why wouldn't it ? 🤔

Reply:
ok.

And why no f-string ?

Reply:
```_("Unable to connect to ElasticSearch:") + f"\n\n{e!r}"``` would be the same, indeed.. but I find it less readable.
Is that what you mean?

Reply:
I mean
```py
_(f"Unable to connect to ElasticSearch:\n\n{repr(e)})"
```
I do not ask for change, just to know if there reasons to not f-string

Reply:
The reason is it wouldn't be translated. You can't use `_()` on dynamically generated strings